pkg/users/accounts-list.js

```diff
@@ -238,6 +356,8 @@
                 return bitem.sortKey - aitem.sortKey;
             }
 
+            if (idx == 2)
+                return bitem.title - aitem.title;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17996-20221206-122041-ea567d53-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)